urllib3/util/retry.py

```diff
@@ -290,5 +289,17 @@ def __repr__(self):
                     cls=type(self), self=self)
 
 
+def redirect_method(method, status):
```

Comment:
Would it make sense to have this live as a method/staticmethod/classmethod under the `Retry` class? This way people could override it with their own behaviour per-request.